src/WhatsNewDialog.cpp

```diff
@@ -42,24 +46,29 @@ namespace
 enum {
    WhatsNewID_WatchReleaseVideo = wxID_HIGHEST + 1,
    WhatsNewID_GoToMuseHub,
+   WhatsNewID_GoToAudioCom,
 };
 
 const char* WhatsNewURL = "https://audacityteam.org/3.7.0-video";
 const char* ChangeLogURL = "https://support.audacityteam.org/additional-resources/changelog";
-const char* MuseHubURL = "https://www.audacityteam.org/mh-whatsnew";
+//const char* MuseHubURL = "https://www.audacityteam.org/mh-whatsnew";
+const char* MuseHubURL = "https://www.musehub.com/app/ace-studio"; //! TODO AU3: update ACE URL
```

Comment:
@grliszas14 I believe that we should continue to use `www.audacityteam.org/mh-whatsnew` here, and the link will be updated in redirects.
@LWinterberg am I correct?